src/javascript/binary/pages/client.js

```diff
@@ -113,6 +113,17 @@ var hide_account_opening_for_risk_disclaimer = function () {
     }
 };
 
+var toggle_hedging_assets = function() {
```

Comment:
can you rename the variable to toggle_hedging_assets_japan? Just to make it very clear that it is Japan-related.